python/src/deltachat/testplugin.py

```diff
@@ -149,22 +151,21 @@ def get_liveconfig_producer(self):
         if not liveconfig_opt:
             pytest.skip("specify DCC_NEW_TMP_EMAIL or --liveconfig to provide live accounts")
 
-        configlist = []
         if not liveconfig_opt.startswith("http"):
             for line in open(liveconfig_opt):
                 if line.strip() and not line.strip().startswith('#'):
                     d = {}
                     for part in line.split():
                         name, value = part.split("=")
                         d[name] = value
-                    configlist.append(d)
+                    self._configlist.append(d)
 
-            yield from iter(configlist)
+            yield from iter(self._configlist)
         else:
             MAX_LIVE_CREATED_ACCOUNTS = 10
```

Comment:
So this variable is the maximum for M, not N\*M?

Reply:
yes, that's the M